conans/client/packager.py

```diff
@@ -25,13 +25,14 @@ def export_pkg(conanfile, package_id, src_package_folder, package_folder, hook_m
     copier = FileCopier(src_package_folder, package_folder)
     copier("*", symlinks=True)
 
-    copy_done = copier.report(output)
-    if not copy_done:
-        output.warn("No files copied from package folder!")
+    copier.report(output)
```

Comment:
Couldn't this ``copier.report()`` be removed? it is going to report the packaged files later.

Reply:
sorry, I forgot this one.

Reply:
done!